scripts/machine_translation/README.md

```diff
@@ -160,11 +190,56 @@ Test BLEU score with 3 seeds (evaluated via sacre BLEU):
 (test bleu / valid bleu)
 | Subword Model | Seed = 123  |
 |---------------|-------------|
-| yttm          | 26.78/25.96 |
+| yttm          | 26.95/25.85 |
 
 - transformer_wmt_en_de_big
 
 (test bleu / valid bleu)
 | Subword Model | Seed = 123  |
 |---------------|-------------|
 | yttm          | 27.99/26.84 |
```

Comment:
I try big NMT. The lr 0.001 will diverge so I decrease it to 0.0005 and the result on testset is 27.818.